roles/servicemesh/tasks/main.yml

```diff
@@ -0,0 +1,377 @@
+---
+- name: Get current state
+  k8s_info:
+    api_version: ripsaw.cloudbulldozer.io/v1alpha1
+    kind: Benchmark
+    name: '{{ meta.name }}'
+    namespace: '{{ operator_namespace }}'
+  register: resource_state
+
+- name: Define variables
+  set_fact:
+    servicemesh: "{{ servicemesh_defaults | combine(workload_args, recursive=True) }}"
+    controlplane_namespace: "{{ meta.name }}-controlplane-{{ trunc_uuid }}"
+    workload_namespace: "{{ meta.name }}-workload-{{ trunc_uuid }}"
+    owner:
+      - apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+        kind: Benchmark
+        name: "{{ meta.name }}"
+        namespace: "{{ operator_namespace }}"
+        uid: "{{ resource_state.resources[0].metadata.uid }}"
+
+- when: resource_state.resources[0].status.state is not defined
+  block:
+  - name: Create control plane namespace
+    k8s:
+      definition:
+        apiVersion: project.openshift.io/v1
+        kind: ProjectRequest
+        metadata:
+          name: "{{ controlplane_namespace }}"
+
+  - name: Create ServiceMesh control plane
+    k8s:
+      definition: "{{ lookup('template', 'smcp.yaml.j2') | from_yaml }}"
+
+  - name: Create workload namespace
+    k8s:
+      definition:
+        apiVersion: project.openshift.io/v1
+        kind: ProjectRequest
+        metadata:
+          name: "{{ workload_namespace }}"
+
+  - name: Create member roll
+    k8s:
+      definition:
+        apiVersion: maistra.io/v1
+        kind: ServiceMeshMemberRoll
+        metadata:
+          name: default
+          namespace: "{{ controlplane_namespace }}"
+        spec:
+          members:
+          - "{{ workload_namespace }}"
+
+  - name: Create Hyperfoil resource
+    k8s:
+      definition:
+        apiVersion: hyperfoil.io/v1alpha2
+        kind: Hyperfoil
+        metadata:
+          name: '{{ meta.name }}-hyperfoil-{{ trunc_uuid }}'
+          namespace: '{{ operator_namespace }}'
+          ownerReferences: "{{ owner }}"
+        spec:
+          # TODO!
+          version: 0.9-SNAPSHOT
```

Comment:
Should this be a var that we default to `0.9-SNAPSHOT` but allow it to be overridden. That way we don't have to rebuild the operator when the version increments?